target/linux/pistachio/Makefile

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+ARCH:=mipsel
+BOARD:=pistachio
+BOARDNAME:=IMG MIPS Pistachio
+FEATURES:=targz nand squashfs ubifs audio ramdisk fpu usb usbgadget
+SUBTARGETS=marduk
```

Comment:
Do you have any plans to add more subtargets soon? If not, you should drop this line.